plugins/inventory/now.py

```diff
@@ -554,11 +571,21 @@ def _get_instance(self):
         instance_env = self._get_instance_from_env()
         return self._merge_instance_config(instance_config, instance_env)
 
+    def _warn_about_deprecations(self):
+        for opt in ("ansible_host_source", "named_groups", "group_by"):
+            if self.get_option(opt):
+                self.display.warning(
+                    "'{0}' options is deprecated since version 1.2.0 and will be "
```

Comment:
Possibly a mix up of singular and plural.
```suggestion
                    "'{0}' options are deprecated since version 1.2.0 and will be "
```

Reply:
Yep, I messed this up, but I need it to be singular here because we print one message per deprecated option.